pkg/templates/kubeadm/v1beta1/kubeadm.go

```diff
@@ -69,6 +69,8 @@ func NewConfig(s *state.State, host kubeoneapi.HostConfig) ([]runtime.Object, er
 			"read-only-port":      "0",
 			"rotate-certificates": "true",
 			"cluster-dns":         nodelocaldns.VirtualIP,
+			// default path is not writable on coreos
+			"volume-plugin-dir": "/etc/kubernetes/kubelet-plugins",
```

Comment:
Let's use /var/lib

```suggestion
			"volume-plugin-dir": "/var/lib/kubelet/volumeplugins",
```

Reply:
Can we use this folder on all supported distros? Otherwise we have to bring knowledge about the used control-plane distros to this template?